src/devices/bus/coco/cococart.cpp

```diff
@@ -48,6 +48,18 @@
 #define LOG_LINE                0
 
 
+/***************************************************************************
+	CONSTANTS
+***************************************************************************/
+
+enum
+{
+	TIMER_CART,
+	TIMER_NMI,
+	TIMER_HALT
+};
```

Comment:
If there's any chance your device could be subclassed, timer IDs should be in the header so derived classes have some chance of not using conflicting IDs.